src/luaotfload-auxiliary.lua

```diff
@@ -477,8 +512,36 @@ function aux.provides_language(font_id, asked_script, asked_language)
     return false
   end
   asked_script   = stringlower(asked_script)
-  asked_language = stringlower(asked_language)
-  if font_id and font_id > 0 then
+  local tfmdata = identifiers[font_id]
+  if not tfmdata then
+    logreport ("log", 0, "aux", "no font with id %d", font_id)
+    return false
+  end
+  local hbface = get_hbface(tfmdata)
+  if hbface then
+    asked_language  = stringupper(asked_language)
+    if asked_language == "DFLT" then
+      return aux.provides_script(font_id, asked_script)
+    end
+    local script = harf.Tag.new(asked_script)
+    -- HarfBuzz expects "DFLT" in lower-case
```

Comment:
Not HarfBuzz but the OpenType spec, there is no `dflt` script tag.

Reply:
https://docs.microsoft.com/en-us/typography/opentype/spec/scripttags

Reply:
@khaledhosny  Right, the comment moved to the wrong location, it was supposed to be above `asked_language` where the specs do not name the default.